### PR TITLE
Misc fixes

### LINKED ISSUE #67: Necromorphs can check pulse and attempt CPR on humans
#### Description of issue
When on help intent as a Necro, will attempt CPR and/or check pulse on a human

#### Client version, Server revision & Game ID

Latest Dev branch

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of the issue being looked at sooner.

- [ ] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [ ] Issue could be reproduced in multiple rounds
- [ ] Issue happened in a recent (less than 7 days ago) round
- [ ] [Couldn't find an existing issue about this](https://github.com/Baystation12/Baystation12/issues)

### PR DESCRIPTION
**Pull request by JakeDaBoss**
Fixes a bunch of previously broken things. Yay.

Closes #67

### CODE REVIEW

File: code/datums/craft/recipes/floor.dm [hunk 1]
```diff
@@ -1,254 +1,119 @@
...
+// WOOD
 
-/datum/craft_recipe/floor/dark/violetcorener
-	name = "dark violet corener tile"
-	result = /obj/item/stack/tile/floor/dark/violetcorener
+/datum/craft_recipe/floor/wood
+	name = "wood floor tile"
+	result = /obj/item/stack/tile/wood
+	steps = list(
+		list(CRAFT_MATERIAL, 1, MATERIAL_WOOD)
```
Comment (NanakoAC) [inline code/datums/craft/recipes/floor.dm]: These need to go:
CRAFT_MATERIAL, materialname, quantity, worktime

you got it right previously, its ordered wrong here

**Review by NanakoAC — CHANGES_REQUESTED** [review]
one change needed

File: code/datums/craft/recipes/floor.dm (hunk 1 above)
Comment (JakeDaBoss) [reply code/datums/craft/recipes/floor.dm]: Oops.

**Review by NanakoAC — APPROVED** [review]

**Comment by JakeDaBoss** [comment]
A mention to https://github.com/DS-13-Dev-Team/DS13/pull/596/commits/63748c7d8465f29cf84c63cc4acc2c92de9ce795 : It disables the command, but does not remove it. But apparently it's not meant to, since it appears for ghosts too.

**Comment by JakeDaBoss** [comment]
@Snypehunter007 can be merged. I decided to do it another way to disable verbs for Necro's. Coming in a megafix later.

**Pull request merged**